app/products/calls/screens/host_controls/host_controls.tsx

```diff
@@ -43,19 +43,40 @@ export const HostControls = ({
     const intl = useIntl();
     const {bottom} = useSafeAreaInsets();
 
+    const sharingScreen = currentCall.screenOn === session.sessionId;
```

Comment:
Unrelated just find it a bit confusing as the variable name suggests a boolean when in fact it's the session id of the presenter.

Reply:
Totally agree, I'm confused too. I think it was a leftover from an old naming scheme "who is the sceenOn" maybe?